packages/aws-cdk/lib/api/hotswap/common.ts

```diff
@@ -65,38 +76,6 @@ export enum HotswapMode {
   FULL_DEPLOYMENT = 'full-deployment',
 }
 
-/**
- * Represents a change that can be hotswapped.
- */
-export class HotswappableChangeCandidate {
```

Comment:
This is a class, but was never used as one. It's now an interface as it should be.